app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,84 @@
+body {
```

Comment:
We shouldn't need to introduce new CSS for this feature. We should be able to leverage existing CSS and components.